README.md

```diff
@@ -126,27 +126,40 @@ Tag\|TagB                    |have either TagA or TagB.
   -m, --machine-readable    | Prints output in JSON format
   -v, --version             | Print Gauge and plugin versions
 
-## Run openshift-pipeline tests
+## Run triggers tests
```

Comment:
this command runs both pipelines and triggers tests

Reply:
oh, yeah sorry I missed it